### DISCUSSION TITLE
How the UITests could be important? is it worth to integrate maestro?

### DISCUSSION THREAD

**Discussion by ghadeermohameedelmahdy** [mobile-dev-inc/Maestro#2066] [Q&A]
I couldn't imagine how the ui tests important like the unit testing, so if anyone could share with us how it was helpful to him or to business needs, thanks in advance.

**Comment by idlework** [comment]
@ghadeermohameedelmahdy There is development in this area. You can follow this in this [PR](https://github.com/mobile-dev-inc/maestro/pull/2078)
